README.rst

```diff
@@ -0,0 +1,415 @@
+.. raw:: html
+
+    <p align="center"><img src="https://user-images.githubusercontent.com/16640218/34506318-84d0c06c-efe0-11e7-8831-0425772ed8f2.png" alt="Logo" width="200"/></p>
+    <br/>
+
+Horovod
+=========
+
+.. image:: https://badge.buildkite.com/6f976bc161c69d9960fc00de01b69deb6199b25680a09e5e26.svg?branch=master
+   :target: https://buildkite.com/horovod/horovod
+   :alt: Build Status
+
+.. image:: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+   :target: https://img.shields.io/badge/License-Apache%202.0-blue.svg
+   :alt: License
+
+.. image:: https://app.fossa.com/api/projects/git%2Bgithub.com%2Fhorovod%2Fhorovod.svg?type=shield
+   :target: https://app.fossa.com/projects/git%2Bgithub.com%2Fhorovod%2Fhorovod?ref=badge_shield
+   :alt: FOSSA Status
+
+.. image:: https://bestpractices.coreinfrastructure.org/projects/2373/badge
+   :target: https://bestpractices.coreinfrastructure.org/projects/2373
+   :alt: CII Best Practices
+
+.. image:: https://pepy.tech/badge/horovod
+   :target: https://pepy.tech/project/horovod
+   :alt: Downloads
+
+.. inclusion-marker-start-do-not-remove
+
+.. contents::
+
+
+Horovod is a distributed training framework for TensorFlow, Keras, PyTorch, and MXNet. The goal of Horovod is to make
+distributed Deep Learning fast and easy to use.
+
+.. image:: https://github.com/LFDLFoundation/artwork/raw/master/lfdl/horizontal/color/lfdl-horizontal-color.png
```

Comment:
Can we make this logo smaller?  On my preview, it takes up the whole width.

Reply:
Solved, thanks!